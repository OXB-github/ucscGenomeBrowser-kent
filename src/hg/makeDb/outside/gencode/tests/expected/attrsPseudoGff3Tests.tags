ENST00000422009.5	basic
ENST00000422009.5	overlapping_locus
ENST00000450773.1	basic
ENST00000450773.1	overlapping_locus
