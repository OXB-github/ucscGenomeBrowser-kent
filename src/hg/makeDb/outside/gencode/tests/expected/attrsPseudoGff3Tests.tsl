ENST00000422009.5	5
ENST00000450773.1	NA
