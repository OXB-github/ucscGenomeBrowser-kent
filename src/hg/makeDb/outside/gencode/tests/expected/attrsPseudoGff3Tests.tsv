geneId	geneName	geneType	unused1	transcriptId	transcriptName	unused2	unused3	unused4	transcriptType	ccdsId	level	transcriptClass	proteinId
ENSG00000242507.6	FO393401.1	transcribed_unprocessed_pseudogene		ENST00000422009.5	FO393401.1-201	processed_transcript					2	pseudo	
ENSG00000242507.6	FO393401.1	transcribed_unprocessed_pseudogene		ENST00000450773.1	FO393401.1-202	transcribed_unprocessed_pseudogene					2	pseudo	
